### PR TITLE
docs/charon: add docs for `combine` command

### PR DESCRIPTION
**Pull request by gsora**
## Summary
Adds documentation for the `combine` charon command.

ticket: #123

### CODE REVIEW

File: docs/int/quickstart/quickstart-combine.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+---
+sidebar_position: 2
+description: Combine distributed validator keyshares
+---
+
+# Combine distributed validator keyshares into an Ethereum validator key
```
Comment (thomasheremans) [inline docs/int/quickstart/quickstart-combine.md:6]:
```suggestion
# Combine DV keyshares

Combine distributed validator keyshares into an Ethereum validator key
```

File: docs/int/quickstart/quickstart-combine.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
+---
+sidebar_position: 2
```
Comment (thomasheremans) [inline docs/int/quickstart/quickstart-combine.md:2]:
```suggestion
sidebar_position: 5
```

**Review by thomasheremans — APPROVED** [review]
Left a few suggestions to improve leftbar navigation but otherwise LGTM

**Comment by OisinKyne** [comment]
Hey @gsora 

Didn't we change the output of `combine` to produce one `validator_keys` folder with every keystore in it? Is this pull request stale and do we need to update this bit?  https://github.com/ObolNetwork/obol-docs/pull/134/files#diff-550469a18b30f575b85b9b619bce05e06b5b895fc19503de8ac8fd62285f7b41R286

**Comment by gsora** [comment]
No, it creates one directory for each validator, named after its public key, with a directory called validator_keys and the keystore in it.

**Comment by OisinKyne** [comment]
> No, it creates one directory for each validator, named after its public key, with a directory called validator_keys and the keystore in it.

Did you not change that output format though after making this comment? https://github.com/ObolNetwork/charon/pull/1799#issuecomment-1439985192

**Comment by gsora** [comment]
Well yes I did do what the comment suggested, moving the keystore files to a validator_keys directory. 

I still retained the “one validator/directory approach” though. 

Considering code is already merged, if this behavior is not what you want please open a issue and let’s get this sprint’d :-)

**Pull request merged**